### ISSUE TITLE
change langage and format

### ISSUE THREAD

**Issue by yohnan3396** [Sharinglabs/cordova-plugin-datepicker#16]
hi,

it is possible to change the format of the date and language ? 

thanks
regards

**Comment by Heshyo** [comment]
No, it uses the language of the phone (which is what the user would expect).

There should be ways to add a parameter to modify that. Eg in Android a quick search led me to this following stackoverflow question: http://stackoverflow.com/q/21257014/276648 Something similar should exist in iOS too.

**Comment by yohnan3396** [comment]
Ok, thanks for the link

Last question, how put the dd/mm/yyyy format when we click on a date ??
